supertokens_python/recipe/passwordless/interfaces.py

```diff
@@ -173,6 +173,21 @@ def __init__(self, status: Literal['OK', 'UNKNOWN_USER_ID_ERROR',
         self.status = status
 
 
+class DeleteUserInfoResult(ABC):
+    def __init__(self, status: Literal['OK', 'UNKNOWN_USER_ID_ERROR']):
+        self.status = status
+
+
+class DeleteUserInfoOkResult(DeleteUserInfoResult):
+    def __init__(self):
+        super().__init__('OK')
+
+
+class DeleteUserInfoUnknownUserIdErrorResult(DeleteUserInfoResult):
+    def __init__(self):
+        super().__init__('UNKNOWN_USER_ID_ERROR')
```

Comment:
Please be consistent with other classes. You need a `self.is_*` boolean. I know that it doesn't serve much purpose, but removing that `is_*` boolean from all such classes is another issue. So for now, we should be consistent.

Reply:
Okay. Done